user-guide/docs/usecases/padgett/usecase_JN_viz.md

```diff
@@ -1,48 +1,76 @@
-# Visualization of spatially distributed data 
-## Jupyter notebook for visualization of spatially distributed data in risk and resilience analysis
+## Visualization of spatially distributed data 
+
+<span style="font-size:1.5em;">
+**Jupyter notebook for visualization of spatially distributed data in risk and resilience analysis** 
+</span> 
+
+
 **Raul Rincon - Dept. of Civil and Environmental Engineering, Rice University** <br>
 **Jamie E. Padgett - Dept. of Civil and Environmental Engineering, Rice University** <br>
 <br>
 _Keywords: visualization; risk and resilience; infrastructure systems; static, interactive, and animated maps and figures, effective communication_
 
-## Resources 
-The examples presented make use of the following DesignSafe resources: <br>
-Jupyter notebooks on DS Juypterhub [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/hub/user-redirect/lab/tree/NHERI-Published/PRJ-3939/visualization_risk_resilience.ipynb)
+### Resources
+ 
+#### Jupyter Notebooks
+The following Jupyter notebook is the basis for the use case described in this section. You can access and run it directly on DesignSafe by clicking on the "Open in DesignSafe" button.
+
+| Scope | Notebook |
+| :-------: | :---------:  |
+| visualization of <br> spatially-distributed Data | visualization_risk_resilience.ipynb <br> [![Open In DesignSafe](https://raw.githubusercontent.com/geoelements/LearnMPM/main/DesignSafe-Badge.svg)](https://jupyter.designsafe-ci.org/user/silvia/lab/workspaces/auto-k/tree/NHERI-Published/PRJ-3939v3/visualization_risk_resilience.ipynb)|
+
+
+
+#### DesignSafe Resources
+The following DesignSafe resources were used in developing this use case.
+
+* [Jupyter Notebook on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){:target="_blank"} <br/> 
```

Comment:
<details><summary>🤔 What is the intent of the <code>&lt;br/&gt;</code>? The end of the list item and the list should already achieve a new line.</summary>

```suggestion
* [Jupyter Notebook on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){:target="_blank"}
```

</details>

Reply:
| with `<br/>`<br><sup>has no effect</sup> | sans `<br/>`<br><sup>renders the same</sup> |
| - | - |
| <img width="1200" alt="usecase_JN_viz with br (has no effect)" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/e85e74c6-fce8-4395-af57-ee37748ef503"> | <img width="1200" alt="usecase_JN_viz sans br (renders the same)" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/6398151d-5693-4fe8-9e79-8bd166adce40"> |